### DISCUSSION TITLE
What relative sequence number do?

### DISCUSSION THREAD

**Discussion by Snalico** [crunchy-labs/crunchy-cli#346] [Q&A]
Hi, 

I have a question regarding relative numbers.
I am using this command:
archive -r %resolution% --skip-existing --ffmpeg-preset %ffmpeg% -a %audio% -s %subtilte% --default-subtitle %subtilte% -o "{series_name}\Season {season_number}\{series_name} - S{season_number}E{relative_sequence_number} - {title}.mkv" %url%
wirth url=https://www.crunchyroll.com/fr/series/GDKHZEJ0K/solo-leveling[S1E6-S1E9]

In the command prompt, I could see the increment number is not relative:

::      8. Title 8 S01E7.5
::      9. Title 9  S01E08

and thus even though during the downloading step it is written: S01E7.5, my file will be written as  S01E8, and the next one will be 9 instead of 8.
I can reproduce it with solo leveling.

Question: is there a way for me to be able to have 7.5 and not 8 to avoid the increment on the classic episode? 

Edit: To note I don't seem to reproduce the issue with mashle, so maybe it is solo leveling specific?

**Comment by khanv1ct** [comment]
This happened to me with Solo Leveling as well, I guess there's something weird about that series page or something. Other series like My Hero Academia look fine so far (output below).

> My Hero Academia (2017) - S02E01 - 13.5 - Hero Notebook [1080P].mkv
